ScoutSuite/providers/gcp/resources/iam/keys.py

```diff
@@ -0,0 +1,22 @@
+from ScoutSuite.providers.gcp.facade.gcp import GCPFacade
+from ScoutSuite.providers.base.configs.resources import Resources
+
+class Keys(Resources):
+    def __init__(self, gcp_facade: GCPFacade, project_id: str, service_account_email: str):
+        self.gcp_facade = gcp_facade
+        self.project_id = project_id
+        self.service_account_email = service_account_email 
+
+    async def fetch_all(self):
+        raw_keys = await self.gcp_facade.iam.get_keys(self.project_id, self.service_account_email)
+        for raw_key in raw_keys:
+            key_id, key = self._parse_key(raw_key)
+            self[key_id] = key
+
+    def _parse_key(self, raw_key):
+        key_dict = {}
+        key_dict['id'] = raw_key['name'].split('/')[-1]
```

Comment:
Can we add a small comment here explaining where this comes from?

Reply:
Done!